app/src/main/java/org/oppia/app/profile/ProfileChooserFragmentPresenter.kt

```diff
@@ -247,4 +253,13 @@ class ProfileChooserFragmentPresenter @Inject constructor(
       )
     }
   }
+
+  private fun logProfileChooserEvent(){
```

Comment:
single space will come before `{`. Also, there is 2 unused import on line number 4 and 17, best to remove those as they don't need in the file. 
Try running ` ktlint --android "app/src/main/java/org/oppia/app/profile/ProfileChooserFragmentPresenter.kt"` and fix these 3 small issue in this file. 
Let me know if you find any difficulty in ktlint. We are using ktlint version 0.37.

Reply:
Done.